### PR TITLE
Update Ignition spec 2.2.0 configs; clean up some configs

### PR DESCRIPTION
**Pull request by bgilbert**
- Update remaining Ignition spec 2.2.0 configs to 3.1.0
- Drop obsolete or empty fields
- Sort some sections into a more logical field order
- Convert decimal file modes to octal
- Drop media type from data URLs
- Unquote file path
- Consistently use `overwrite: true` in SCTP docs

cc @bobfuru @miabbott

### CODE REVIEW

**Comment by bobfuru** [comment]
LGTM!

File: modules/cnf-installing-the-operators.adoc [hunk 1]
```diff
@@ -157,21 +157,17 @@ metadata:
...
+          contents:
             source: data:,
-            verification: {}
-          filesystem: root
-          mode: 420
-          path: /etc/modprobe.d/sctp-blacklist.conf
-        - contents:
+        - path: /etc/modules-load.d/sctp-load.conf
+          mode: 0644
+          contents:
```
Comment (miabbott) [inline modules/cnf-installing-the-operators.adoc:169]: In `modules/nw-sctp-enabling.adoc` there is a similar section about configuring SCTP and the MC uses `overwrite: true` for the `.conf` files.

I think we should do the same here for consistency, as this also discusses enabling SCTP

**Review by miabbott — APPROVED** [review]
One nit, but otherwise LGTM

**Comment by bgilbert** [comment]
Updated, including a few more small cleanups.

**Pull request merged**